kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -1,10 +1,24 @@
 # (C) Datadog, Inc. 2023-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+from confluent_kafka.admin import AdminClient
+
 from datadog_checks.kafka_consumer.client.kafka_client import KafkaClient
 
 
 class ConfluentKafkaClient(KafkaClient):
+    @property
+    def kafka_client(self):
+        if self._kafka_client is None:
+            self._kafka_client = AdminClient(
+                {
+                    "bootstrap.servers": self.config._kafka_connect_str,
+                    "socket.timeout.ms": self.config._request_timeout_ms,
+                    "client.id": "dd-agent",
+                }
+            )
+        return self._kafka_client
```

Comment:
To reviewer: this `kafka_client` is very barebones and likely won't work for TLS or Kerberos connections. However, I think we should start with something like this and add TLS/Kerberos in their own PRs with new tests.